templates/pages/brands.html

```diff
@@ -29,6 +30,7 @@ <h4 class="card-title">
         {{/each}}
     </ul>
     {{> components/common/paginator pagination.brands}}
+    {{{region name="page_content"}}}
```

Comment:
Could these be better named to give some indication of positioning? "Top" vs "bottom" etc?